tests/conftest.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+import json
+import apypie
+import requests
+
+from collections import namedtuple
+
+User = namedtuple('User', ['username', 'password', 'name'])
+
+
+def pytest_generate_tests(metafunc):
+    with open(metafunc.config.option.variables[0]) as f:
+        variables = json.load(f)
+    assert variables
+
+    if 'url' in metafunc.fixturenames:
+        base_url = f"https://{variables['server_hostname']}"
+        assert base_url
```

Comment:
This will always be true since `base_url` is always a non-empty string. It have raised an exception on the line before, but if it gets here, it'll be truthy.